Loop/Views/Presets/Training Content/Components/TherapySettingsExampleView.swift

```diff
@@ -116,7 +116,7 @@ struct TherapySettingsExampleView: View {
         case .carbRatio(let double):
             Text("\(numberFormatter.string(from: double) ?? "0")").bold() + Text("\u{00a0}\(LoopUnit.gramsPerUnit.shortLocalizedUnitString())")
         case .isf(let double):
-            Text(displayGlucosePreference.format(LoopQuantity(unit: .milligramsPerDeciliter, doubleValue: double), includeUnit: false)).bold() + Text("\u{00a0}\(displayGlucosePreference.unit.shortLocalizedUnitString())")
```

Comment:
These 3 bottom file changes are design changes for presets training. Unrelated to LOOP-5393.